tools/cli/air-trace/src/utils.rs

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+use std::time::Duration;
+
+pub(crate) fn parse_tracing_duration(input: &str) -> Result<Duration, anyhow::Error> {
```

Comment:
Are there any dedicated crate for that? Looks like smth like https://docs.rs/parse_duration/latest/parse_duration/ could suit (but I haven't look much to its API).

Reply:
I'v tried several ones, but one does not support floats, another does not support suffixes used by the output format, etc.